GarNet/garnet.py

```diff
@@ -303,19 +245,25 @@ def construct_garnet_file(reference_file, motifs_file, output_file, options):
 		motif_genes_df (pd.DataFrame): motif-gene associations and distance between the two. 
 	"""
 
+	# Check whether motif_file_or_files is single path or list of paths
+	if isinstance(motif_file_or_files, list): motif_files = motif_file_or_files
+	else: motif_files = [motif_file_or_files]
```

Comment:
I think it makes sense to take the extra step and validate that everything in the list is a path -- take a look at the way this is done in `map_peaks`. Seem reasonable?

Reply:
Yahp good point.

Reply:
0a917cad0d9d5ebb1f65c7e38bac35e2b50c9fb6